### PR TITLE
Delete deprecated old propagation model

### PR DESCRIPTION
**Pull request by SergeAstapov**
Part of #116.

Follow up to #499.

### CODE REVIEW

File: tests/dummy/app/services/keyboard.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { action } from '@ember/object';
```
Comment (SergeAstapov) [inline tests/dummy/app/services/keyboard.js:1]: this allows us to keep documentation and interactive widgets/examples for 6.x

Basically deprecated/removed functionality for old propagation model/priority based responders moved from add-on code to dummy app for the purpose of interactive docs only.

File: tests/dummy/app/mixins/enterable.js [hunk 2]
```diff
@@ -13,10 +14,22 @@ export default Mixin.create( {
...
 
   _activate() {
     this.widgetManager.activate(this);
     this.set('keyboardActivated', true);
+
+    // Support old propagation model (< 7.x) in docs.
+    // See tests/dummy/app/services/keyboard.js which extends addon service with old code.
+    // See tests/dummy/app/templates/priority.hbs when property is set usage.
+    if (this.oldPropagationModel) {
+      this.keyboard.isPropagationEnabled = false;
```
[the remark this answers is not in the document]
Comment (SergeAstapov) [reply tests/dummy/app/mixins/enterable.js:32]: see `tests/dummy/app/services/keyboard.js` for the usage

File: tests/dummy/app/templates/application.hbs [hunk 3]
```diff
@@ -2,7 +2,8 @@
   <nav class="menu">
     <LinkTo @route='index'>Overview</LinkTo>
     <LinkTo @route='usage'>Usage</LinkTo>
-    <LinkTo @route='priority'>Priority</LinkTo>
+    <LinkTo @route='event-propagation'>Event propagation</LinkTo>
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/application.hbs:5]: new docs page added to navigation

File: tests/dummy/app/templates/components/keyboard-propagation-widget.hbs [hunk 4]
```diff
@@ -1,7 +1,11 @@
-{{format-markdown "
-### Event propagation
+{{#if @v6Intro}}
+  {{format-markdown "
+  ### Event propagation
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/components/keyboard-propagation-widget.hbs:3]: there is no need to duplicate page title

File: tests/dummy/app/templates/event-propagation.hbs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{{format-markdown "
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/event-propagation.hbs:1]: New docs page is very similar to the previous "Priority" but without things removed in 7.x

File: tests/dummy/app/services/keyboard.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+/**
+ * Support old propagation model (< 7.x) in docs.
+ * See tests/dummy/app/mixins/enterable.js for usage.
+ */
+export default class extends KeyboardService {
+  isPropagationEnabled = true;
+
+  @action
+  _respond(event) {
+    if (this.isPropagationEnabled) {
```
Comment (SergeAstapov) [inline tests/dummy/app/services/keyboard.js:70]: this is the switch for "old code path" which may be initiated from `tests/dummy/app/mixins/enterable.js`

File: tests/dummy/app/templates/application.hbs [hunk 7]
```diff
@@ -2,7 +2,8 @@
...
+    <LinkTo @route='priority'>Priority (&lt;= 6.x)</LinkTo>
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/application.hbs:6]: Explicitly marked for which version it focuses on.

File: tests/dummy/app/templates/priority.hbs [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {{format-markdown "
-## Priority
+## Priority (6.x and below)
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/priority.hbs:2]: Keep this page with documentation and interactive widgets/examples for 6.x

Basically deprecated/removed functionality for old propagation model/priority based responders moved from add-on code to dummy app for the purpose of interactive docs only.

File: tests/dummy/app/templates/upgrading.hbs [hunk 9]
```diff
@@ -26,4 +26,16 @@ and required the use of `code` identifiers, like `KeyT`, `Digit1`, etc. Be sure
 [docs regarding `key` vs. `code` properties](usage) and be sure you are specifying the key
 combo consistent with your intended behavior. You will likely find that you want to update some key
 combos to the `key` equivalent. e.g. replace `ctrl+KeyT` with `ctrl+t`.
+
+### Upgrading from ember-keyboard 6.x to 7.x
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/upgrading.hbs:30]: Started new section. Does not seem we need to be much more explicit.

**Review by lukemelia — APPROVED** [review]
Fantastic work! 🙌

**Pull request merged**